src/sql/workbench/contrib/notebook/browser/notebookViews/notebookviews.css

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.grid-stack {
+	margin: 0 8px;
+}
+.grid-stack>.grid-stack-item {
```

Comment:
Add space before and after `>`
`.grid-stack > .grid-stack-item`
Applies to all instances below.